### PR TITLE
Reddit testing with PRAW

### PR DESCRIPTION
**Pull request by ajn1390**

### CODE REVIEW

**Comment by chumomega** [comment]
can you run `pip freeze > requirements.txt` so we can track the new dependency you may have installed locally?

File: server/model/RedditTest.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import praw
+import os
+from praw.models import MoreComments
+from dotenv import load_dotenv
+
+
+reddit = praw.Reddit("test") # praw.ini file has credetials
+reddit.read_only = True
+print(reddit.read_only)
+
```
Comment (chumomega) [inline server/model/RedditTest.py:12]: can you turn this into a class that implements the methods git chamber and get chamber members? 
see https://github.com/chumomega/echo_chamber/blob/main/server/model/YoutubeClient.py for example

**Comment by chumomega** [comment]
can you update Chamber type so we support reddit now https://github.com/chumomega/echo_chamber/blob/main/server/model/ChamberType.py

**Comment by chumomega** [comment]
check this out https://github.com/chumomega/echo_chamber/blob/main/server/controllers/getEchoChamberStatus.py#L51
if we want to support reddit, we'll need to update.

then you can test it out by calling this endpoint with the identifier and chamber type https://github.com/chumomega/echo_chamber/blob/main/server/controllers/getEchoChamberStatus.py#L21

File: server/model/RedditTest.py (hunk 1 above)
Comment (ajn1390) [reply server/model/RedditTest.py:12]: If we're going to have those methods in multiple places...should we consider a base class?

Comment (chumomega) [reply server/model/RedditTest.py:12]: yup!

**Pull request closed without merging**